src/EditorFeatures/TestUtilities/RefactoringHelpers/RefactoringHelpersTestBase.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.UnitTests.RefactoringHelpers
+{
+    [UseExportProvider]
+    public abstract class RefactoringHelpersTestBase<TWorkspaceFixture> : TestBase, IClassFixture<TWorkspaceFixture>, IDisposable
+        where TWorkspaceFixture : TestWorkspaceFixture, new()
+    {
+        protected TWorkspaceFixture fixture;
```

Comment:
readonly?

Reply:
Copied from InferrerHelperTestBase, should I fix it there as well?

Reply:
Fixed in both places.

Reply:
yup yup!